bin/dsn.cmake

```diff
@@ -572,11 +572,11 @@ function(dsn_common_setup)
     endif()
     
     message (STATUS "Installation directory: CMAKE_INSTALL_PREFIX = " ${CMAKE_INSTALL_PREFIX})    
-    set(DSN_ROOT2 "$ENV{DSN_ROOT}")
-    if((NOT (DSN_ROOT2 STREQUAL "")) AND (EXISTS "${DSN_ROOT2}/"))
-        set(CMAKE_INSTALL_PREFIX ${DSN_ROOT2} CACHE STRING "" FORCE)
-        message (STATUS "Installation directory redefined w/ ENV{DSN_ROOT}: " ${CMAKE_INSTALL_PREFIX})    
-    endif()
+    #set(DSN_ROOT2 "$ENV{DSN_ROOT}")
```

Comment:
this actually breaks the connection between rDSN and rDSN's applications. Our code generator assumes that it can find the libs and includes etc. through env variable DSN_ROOT.